Backend/BikesRentalServer.Services/UsersService.cs

```diff
@@ -25,7 +25,7 @@ public UsersService(IUsersRepository usersRepository, IReservationsRepository re
 
         public ServiceActionResult<IEnumerable<User>> GetAllUsers()
         {
-            var users = _usersRepository.GetAll();
+            var users = _usersRepository.GetAll().Where(u => u.Role == UserRole.User);

```

Comment:
Jeśli okazał się być potrzebny to sprawdź testy dla `GetAllUsers` - pewnie jest tam test analogiczny jak ten co napisałeś (czyli niepoprawny). Wywal go wtedy i usuń tego where'a

Reply:
usunąłam te testy

Reply:
chociaż normalnie nie lepiej by je było naprawić? XD